docusaurus.config.js

```diff
@@ -71,29 +71,16 @@ const config = {
 			footer: {
```

Comment:
Footer seems rather misaligned to me here. I propose not to center the copyright, at least on desktop.

<img width="1350" alt="image" src="https://user-images.githubusercontent.com/375537/225033741-b0f7bc87-10e2-4b9f-b7d3-571282c185d2.png">